timescaledb/how-to-guides/install-timescaledb/installation-windows.md

```diff
@@ -1,59 +1,43 @@
-## Windows ZIP Installer [](installation-windows)
-
-**Note: TimescaleDB requires PostgreSQL 12, 13, or 14.**
-
-#### Prerequisites
-
-- [Visual C++ Redistributable for Visual Studio 2015][c_plus] (included in VS 2015 and later)
-- A standard **PostgreSQL 12, 13, or 14 64-bit** installation
-- Make sure all relevant binaries are in your PATH: (use [pg_config][])
-- Installation must be performed from an account with admin privileges
-
-#### Build and install
-
-1. Download the the .zip file for your PostgreSQL version - [12][windows-dl-12] or [13][windows-dl-13] or [14][windows-dl-14].
-1. Extract the zip file locally
-1. Run `setup.exe`, making sure that PostgreSQL is not currently running
-1. If successful, a `cmd.exe` window will pop open and you will see the following:
-
-```bash
-TimescaleDB installation completed succesfully.
-Press ENTER/Return key to close...
-```
-Go ahead and press ENTER to close the window
-
-#### Updating from TimescaleDB 1.x to 2.0
-Once the latest TimescaleDB 2.0 are installed, you can update the EXTENSION
-in your database as discussed in [Updating Timescale to 2.0][update-tsdb-2].
-
-#### Configure your database
-
-There are a [variety of settings that can be configured][config] for your
-new database. At a minimum, you will need to update your `postgresql.conf`
-file to include `shared_preload_libraries = 'timescaledb'`.
-If you ran `timescaledb-tune` during the install, you are already done.
-If you did not, you can re-run the installer.
-
-This will ensure that our extension is properly added to the parameter
-`shared_preload_libraries` as well as offer suggestions for tuning memory,
-parallelism, and other settings.
+## Install self-hosted TimescaleDB on Windows systems
+You can host TimescaleDB yourself on your Microsoft Windows system.
+These instructions use a `zip` installer on these versions:
+*   Microsoft Windows 10
+*   Microsoft Windows 11
+*   Microsoft Windows Server 2019
```

Comment:
Hmm i'm no expert on the windows compatibility i think we should probably run on any Windows x86-64

Reply:
Who should I ask?